### DISCUSSION TITLE
How can I change text color of antd notification depending upon type?

### DISCUSSION THREAD

**Discussion by sardar-abd** [ant-design/ant-design#36551] [Q&A]
How can I change text color of antd notification depending upon type. I want red color of notification if type is error and white color for other types. I am able to change background colors depending upon type but I dont know why I am unable to change text color.

This is my code that is working for background color, borders etc but I need to change text color when type of antd notification changes.

```
import { notification } from "antd";

const openNotification = (title, message, type) => {
  notification[type]({
    message: title,
    description: message,
    style: { backgroundColor: type == "error" ? "red" : "white" },
    duration: 0,
  });
};

export default openNotification;
```

**Comment by siam88** [comment]
I checked your code it works fine for me. You can check my code too

My Code Example : [Demo](https://stackblitz.com/edit/react-m59eqv?file=demo.js)
